packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2896,3 +3025,270 @@ typedef SelectableRegionContextMenuBuilder = Widget Function(
   BuildContext context,
   SelectableRegionState selectableRegionState,
 );
+
+/// The status of the [SelectionListener].
+enum SelectionListenerStatus {
+  /// Indicates the selection under a [SelectionListener] is changing.
+  changed,
+
+  /// Indicates the selection under a [SelectionListener] is finalized.
```

Comment:
Removed `addStatusListener` from `SelectionListenerNotifier` for now since I think that it makes more sense for it to live on `SelectableRegionScope`.